server/tasks/stream_inference.py

```diff
@@ -71,7 +71,7 @@ async def run_streaming(
     task_time_limit=settings.task_time_limit_sec,
 )
 def streaming_parent_task(
-    self, agent: Agent, result_handler: ResultHandler, batch_size: int = 10
+    self, agent: Agent, result_handler: ResultHandler, batch_size: int = 1
```

Comment:
Any chance you've tested this in a FB with a reasonably large dataset? My only concern here is that we cause congestion in the DB from doing a separate write for each pred